plugins/@grouparoo/spec-helper/src/lib/factories/group.ts

```diff
@@ -1,14 +1,20 @@
 import { loadPath } from "../loadPath";
 import faker from "faker";
-import { Group } from "@grouparoo/core";
+import { Group, GrouparooModel } from "@grouparoo/core";
+import ModelFactory from "./model";
 
-const data = async (props = {}) => {
+const data = async (props: { modelId?: string } = {}) => {
   const { GrouparooModel } = await import(`@grouparoo/core/${loadPath}`);
+  const model =
+    (props.modelId
+      ? await GrouparooModel.findOne({ where: { id: props.modelId } })
+      : await GrouparooModel.findOne()) ??
+    ((await ModelFactory({ id: props.modelId })) as GrouparooModel);
```

Comment:
See my comment in `plugins/@grouparoo/spec-helper/src/lib/factories/destination.ts`